api/v1/swagger.yaml

```diff
@@ -1988,27 +1997,632 @@ paths:
                 type: array
                 items:
                   type: string
-              keywords:
-                description: Describes this generator.
+              tags:
+                type: array
+                items:
+                  type: string
+                  maxLength: 60
+                  pattern: ^[0-9A-Za-z_][0-9A-Za-z_\\-]{1,59}$
+                description: >
+                  Array of tag names. Tag name: Max 60 characters; case
+                  insensitive; allows alpha-numeric characters, underscore (_)
+                  and dash (-). Tag names cannot start with a dash (-).
+              connection:
+                type: object
+                description: Information about the connection.
+              context:
+                type: object
+                description: >-
+                  Variables accessible inside the transform function.
+            required:
+              - name
+              - aspects
+      responses:
+        201:
+          description: >-
+            Created.
+          schema:
+            $ref: "#/definitions/GeneratorResponse"
+        400:
+          $ref: "#/responses/400"
+        403:
+          $ref: "#/responses/403"
+        default:
+          $ref: "#/responses/genericError"
+
+  # ---------------------------------------------------------------------------
+
+  /generators/{key}:
+    x-swagger-router-controller: generators
```

Comment:
are /generator/** swagger definitions from an older commit? The master branch has this already.